docs/resources/hotels.md

```diff
@@ -1,122 +1,258 @@
-destination, compare rates and rooms and get the final price of the stay. Once
-a room and rate are selected, you can help them complete the reservation with
-the Hotel Booking API. Here’s how it works.
+Let's learn how to get started and help your users book the perfect rooms at over 150,000 hotels worldwide.
 
-## Searching Hotels
+!!! information
+    This page has been updated based on `Hotel Search V3` updates since MAY 2022. 
+
+## Search hotels
+
+### Get a list of hotels by location 
+First, Users should be able to search hotels for a given location. the [Hotel List API](https://developers.amadeus.com/self-service/category/hotel/api-doc/hotel-list/api-reference) returns the list of hotels based on a city or a geographic code to answer a question such as **"what are the hotels closed to the city hall?"** `Hotel List API` has 3 endpoints to utilize based on your search criteria. it returns `hotel name`, `location`, and `hotel id` for you to move to the next steps of the hotel search. 
+
+based on the search criteria, you will get the list of `hotelId` with hotel information as below.
+```json
+        {
+            "chainCode": "AC",
+            "iataCode": "PAR",
+            "dupeId": 700169556,
+            "name": "ACROPOLIS HOTEL PARIS BOULOGNE",
+            "hotelId": "ACPARH29",
+            "geoCode": {
+                "latitude": 48.83593,
+                "longitude": 2.24922
+            },
+            "address": {
+                "countryCode": "FR"
+            },
+            "lastUpdate": "2022-03-01T15:22:17"
+        }
+```
+
+#### Search hotels by a city or Geocode 
 
-The API allows you to search for hotels for a given location and time. The
-endpoints are designed from the perspective of a funnel; i.e. to search broadly
-for all hotels, narrow down to a particular hotel and eventually its unique
-offers. A host of parameters allow the user to filter the response as per
-his/her need. This API thus renders enormous capabilities to build solutions
-for travelers not only for the hotels space, but also complement other travel
-scenarios or enrich apps with hotel-related information. 
+You can specify a [IATA city code](https://www.iata.org/en/publications/directories/code-search/) or Geocode to search a more specific area to get the list of hotels. You can customize the request with parameters like radius, chain code, amenities, star ratings, and hotel source. 
 
-### Search hotels by location
+for example, 
 
-The first endpoint `/shopping/hotel-offers` provides a list of available hotels
-in a chosen location, which is defined by a city code or a set of GPS
-coordinates. You can customize the request with parameters like chain name,
-amenities, star ratings, board type and more. 
+With by-city endpoint, Get list of hotels in Paris with a swimming pool and more than 4 stars : 
+```bash
+curl https://test.api.amadeus.com/v1/reference-data/locations/hotels/by-city?cityCode=PAR&amenities=SWIMMING_POOL&ratings=4,5
```

Comment:
something that I changed in the flights category, instead of curl I would prefer to add the method, such as GET or POST. It's much more clear to know what method is the endpoint. What do you think?

Reply:
agree!